### PR TITLE
Fix name spacing use and explination

### PR DESCRIPTION
**Pull request by aturkewi**
Properly explain name spacing and how to use it in readme

Update tests to use `Players::` rather than `Player::`

@pletcher @PeterBell

### CODE REVIEW

**Comment by aturkewi** [comment]
Closes #19

File: README.md [hunk 1]
```diff
@@ -97,13 +97,25 @@ Finally, a board can return values based on its state such as `#full?` when enti
...
 
 Every player subclass will implement a `#move` method that represents how that type of player makes a move in Tic Tac Toe.
 
-##### 'players/human.rb' - `Player::Human`
+##### 'players/human.rb' - `Players::Human`
 
-Define a class `Player::Human` that inherits from `Player`. The human player must implement a `#move` method that takes in a `board` argument and allows a human player to enter a move via the CLI. The method should return the value the user enters. Even though the method accepts a `board` argument, it does not need to use it.
+Define a class `Human` that inherits from `Player`. This class should be namespaced inside the module `Players` because the `human.rb` file is inside the `players/` directory. This keeps our code nice and tidy.
+
+The human player must implement a `#move` method that takes in a `board` argument and allows a human player to enter a move via the CLI. The method should return the value the user enters. Even though the method accepts a `board` argument, it does not need to use it.
```
Comment (pletcher) [inline README.md:108]: Same question as on `solution`: When wouldn't we want to use `board` here? Isn't that bad method design?

Comment (PeterBell) [inline README.md:108]: As per solution - lets do a separate issue/PR for board so we can ship one thing at a time...

**Comment by PeterBell** [comment]
LGTM @aturkewi - want to merge both this and the other PR once you've got rid of the boilerplate "View on learn" link in the other one?

**Comment by aturkewi** [comment]
@PeterBell do we not want the View On Learn links anymore? I didn't add an here, I just brought he solution inline with the master. Should I be removing those links from both?

**Comment by PeterBell** [comment]
Hey @aturkewi just want to make sure we don't have two copies of it . . .

**Pull request merged**